tasks/board.md

```
@@ -8,12 +8,13 @@
 ## 进行中
 | 任务卡 | owner_module | 模型 | 开始日期 |
 |---|---|---|---|
-| T-W0-011 | src/workbench/ · tests/unit/ | GLM-5.3 | 2026-08-19 |
+| T-W5-031 | cmd/ · core/ · api/ · packs/ · registry/ | GLM-5.3 | 2026-08-19 |
 
 ## 待验
 | 任务卡 | PR | Verifier |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)